### PR TITLE
Fixes made as a result of operator testing

### PR DESCRIPTION
**Pull request by phantomjinx**
- Adds support for installation testing using a locally created bundle-index image;
- Removes the `replace` property from the constructed bundle since no upstream bundle has yet been released;
- Fixes warnings when validating the bundle
  - Adds v1alpha1 CR example
  - Adds cpu and memory resource limits
  - Adds minimum kube version
- Fixes the version of the CR installed with `make app`;
- Updates operator `config.yaml` with default values used in standalone installs
  - Updates title, additionalInfo, appName to include 'Online'
  - Adds image reference paths
  - Removes now invalid online section properties
  - Adds showAppName property to display app name in title bar
  - Adds login properties in case form config is used
  - Adds mandatory 'productInfo' empty array. This works around bug in [hawtio/react (#715)](https://github.com/hawtio/hawtio-next/issues/715)

### CODE REVIEW

**Comment by phantomjinx** [comment]
@tadayosi 
I know this may seem an omnibus PR but figured the commits are so closely related due to conducting an e2e bundle install test. If you do want me to separate them out then let me know.

File: bundle/manifests/hawtio-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -153,13 +179,19 @@ spec:
                       fieldPath: metadata.name
                 - name: OPERATOR_NAME
                   value: hawtio-operator
-                image: docker.io/hawtio/operator:1.0.0
+                image: quay.io/phantomjinx/hawtio-operator:1.0.0-202312141045
```
Comment (tadayosi) [inline bundle/manifests/hawtio-operator.clusterserviceversion.yaml]: phantomjinx image?

**Review by tadayosi — APPROVED** [review]
one comment but otherwise looks good to me.

File: bundle/manifests/hawtio-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (phantomjinx) [reply bundle/manifests/hawtio-operator.clusterserviceversion.yaml]: Me Fail!

Comment (phantomjinx) [reply bundle/manifests/hawtio-operator.clusterserviceversion.yaml]: Sorted.

**Pull request merged**